source/tools/themefy.gd

```diff
@@ -7,61 +7,46 @@ func _run():
 	var interface = get_editor_interface()
 	var filesys = interface.get_resource_filesystem()
 	var dir = filesys.get_filesystem_path(PATH)
-	
+	var count = 0
 	for f in dir.get_file_count():
 		var path = dir.get_file_path(f)
 		if not filesys.get_file_type(path) == "Theme":
 			continue
 		var theme = load(path)
+		print("Creating theme for ", path)
```

Comment:
I think we could take rid of these prints as well, couldn't we? Better to avoid spamming the output terminar :sweat_smile: